db/cache/charts.go

```diff
@@ -895,6 +931,9 @@ var chartMakers = map[string]ChartMaker{
 // Chart will return a JSON-encoded chartResponse of the provided type
 // and BinLevel.
```

Comment:
Maybe note in the `Chart` docs that `binString` is only used for charts that are not window-binned.  It would be confusing to see a different bin string used if you were expecting your input to be used.